.github/workflows/canary.yaml

```
@@ -58,17 +58,36 @@ jobs:
-                   deps[pkg.name] = `${npmBaseUrl}/${pkg.tarball}`;
+                   overrides[pkg.name] = `${npmBaseUrl}/${pkg.tarball}`;
                  }
-                 const depsJson = JSON.stringify({ dependencies: deps }, null, 2);
+                 const overridesJson = JSON.stringify({ overrides }, null, 2);
+
+                 // Build agent prompt
+                 const packageList = packagesJson.map(pkg => `- ${pkg.name}`).join('\n');
+                 const agentPrompt = `Add canary overrides to root package.json and run bun install.
+
+version: ${version}
+base url: ${npmBaseUrl}
```

actionlint (1.7.10)

[error] could not parse as YAML: did not find expected key

(syntax-check)